### PR TITLE
Technical Fork

### PR DESCRIPTION
**Pull request by vinodkahuja**
Signed-off-by: Vinod Kumar Ahuja vahuja@unomaha.edu

### CODE REVIEW

**Comment by geekygirldawn** [comment]
I think the DCO is failing due to the format, which should have the email address enclosed by < > like this: 
Signed-off-by: Vinod Kumar Ahuja <vahuja@unomaha.edu>

But you've included the right info, so I'll see if I can manually set it to pass the check :)

File: focus-areas/what/technical-fork.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+###This metric is a release candidate To comment on this metric please see Issue #xx Following a comment period, this metric will be included in the next regular release.
+
```
Comment (geekygirldawn) [inline focus-areas/what/technical-fork.md:2]: Can you create the issue and update this with the issue number before we merge it?

I also think this is missing some punctuation. It looks like there should be a period after candidate and another one after the issue number?

**Review by geekygirldawn — CHANGES_REQUESTED** [review]

File: focus-areas/what/technical-fork.md (hunk 1 above)
Comment (vinodkahuja) [reply focus-areas/what/technical-fork.md:2]: Updated.

Comment (geekygirldawn) [reply focus-areas/what/technical-fork.md:2]: Much better, thank you!

**Pull request merged**